src/store/modules/user/actions.ts

```diff
@@ -77,9 +77,13 @@ const actions: ActionTree<UserState, RootState> = {
 
         const eComStoreResp = await UserService.getEComStores(payload);
         if (eComStoreResp.status === 200 && eComStoreResp.data.docs?.length > 0 && !hasError(eComStoreResp)) {
+          const userPref =  await UserService.getUserPref({
+            'userPrefTypeId': 'SELECTED_BRAND'
+          });
           const stores = eComStoreResp.data.docs
+          const store = stores.find((store: any) => store.productStoreId == userPref.data.userPrefValue)
```

Comment:
Improve variable name `store`. It should based upon what value we are storing in it

Reply:
Improved sir!